back/api/lib/graphql/index.js

```diff
@@ -3,13 +3,19 @@
 const graphql = require('graphql').graphql;
 const Schema = require('./schema');
 
-module.exports = (query) => {
+module.exports = (eventQuery) => {
 
   // patch to allow queries from GraphiQL
   // like the initial introspectionQuery
-  if (query && query.hasOwnProperty('query')) {
-    query = query.query.replace("\n", ' ', "g");
+  let query, variables;
+
+  if (eventQuery && eventQuery.hasOwnProperty('query')) {
+    query = eventQuery.query.replace("\n", ' ', "g");
+  }
+
+  if (eventQuery && eventQuery.variables) {
+    variables = JSON.parse(eventQuery.variables);
   }
 
-  return graphql(Schema, query);
+  return graphql(Schema, query, null, variables);
```

Comment:
That's for the latest version. It has the right parameters for the current version in the blueprint. But yeah a version upgrade is due.